jest.config.js

```diff
@@ -12,10 +12,6 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
   moduleNameMapper: {
     kbar: '<rootDir>/__mocks__/kbar.ts',
-    // ref: https://react-hooks-testing-library.com/installation#being-specific
```

Comment:
no longer required! the `@testing-library/react` now has a `renderHook` which supports v18. there's no plans to update `@testing-library/react-hooks` to v18 so there's little choice really